pyresample/grid.py

```diff
@@ -236,7 +236,7 @@ def get_resampled_image(target_area_def, source_area_def, source_image_data,
                 # First iteration
                 result = next_result
             else:
-                result = np.row_stack((result, next_result))
+                result = np.ma.row_stack((result, next_result))
```

Comment:
This forces the result to be a numpy masked array. Could you do:

```
if i == 0:
    ...
elif isinstance(result, np.ma.MaskedArray):
    result = np.ma.row_stack((result, next_result))
else:
    result = np.row_stack((result, next_result))
```

Reply:
Good point! I had already pushed when I saw your comment :) It is slightly different now

Reply:
Not sure you needed the `core` part, but eh if it works then OK.

Reply:
I'll wait for tests to pass then merge.

Reply:
Indeed, it works both with and without the ``core`` part. Good to know, thanks!